README.md

```diff
@@ -37,8 +37,18 @@ To run the app locally, use:
 bin/dev
 ```
 
-You should now be able to access the application at [http://localhost:3000](http://localhost:3000).
+## Subdomain Setup
 
+The app will work OK with no subdomains, but to really test out the multi-tenant capabilities, you will need to configure subdomains. Every `Organization` has a subdomain field, and that Org will be accessible via `ApplicationController#current_tenant`. 
+
+To configure your local dev env to test subdomains, you will need to [edit your `/etc/hosts` file](https://linuxize.com/post/how-to-edit-your-hosts-file/)
+
+Add the following line:
+
+```text
+127.0.0.1  pet-rescue.local alta.pet-rescue.local rubyforgood.pet-rescue.local
+```
+This will maps the two seed Orgs with the correct subdomains
```

Comment:
```suggestion
This will maps the two seed Orgs with the correct subdomains

```
Just a space between this and the next line

Reply:
Small typo `This will maps` vs `This will map`

Reply:
fixed